packages/lit-dev-content/samples/examples/context-basics/project.json

```diff
@@ -0,0 +1,15 @@
+{
+  "extends": "/samples/base.json",
+  "title": "Basics",
+  "description": "Share data from a provider element with Context.",
+  "section": "@lit-labs/context",
```

Comment:
wdyt about putting this under the section "Managing Data" to match the doc location? The title could be "Context Basics".

Reply:
I love that.